python/pyproject.toml

```diff
@@ -47,6 +47,7 @@ Changelog = "https://github.com/Mirascope/mirascope/releases"
 
 [project.optional-dependencies]
 anthropic = ["anthropic>=0.64.0,<1.0"]
+anthropic-bedrock = ["anthropic[bedrock]>=0.64.0,<1.0"]
```

Comment:
note: i merged a bump to the anthropic deps, when you sync the anthopic version will change to a future version with minor breaking changes to the typing (Omit vs NotGiven). you will need to update anthropic-bedrock to match